src/schema/sources.ts

```diff
@@ -347,13 +374,74 @@ const sourceToGQL = (source: Source): GQLSource => ({
 });
 
 export enum SourcePermissions {
-  View,
-  Post,
-  Leave,
-  Delete,
-  Edit,
+  View = 'view',
+  Post = 'post',
+  PostDelete = 'post_delete',
```

Comment:
I think there is also a option where members can delete their own posts, how will we tackle this?

Reply:
We will just make the necessary adjustments in the `hasPermissionCheck` function once implementation comes. Then it should automatically be in the rightful permissions array.

Reply:
Or do you mean that we do that here, now?